.config/mkdocs.yml

```diff
@@ -68,6 +68,7 @@ nav:
       - common-practices-tools/balance-scores.md
       - common-practices-tools/best-practices-for-managers-and-team-members.md
       - common-practices-tools/contribution/contrib-first.md
+        - common-practices-tools/contribution/prodev-contrib-sprint.md
```

Comment:
I think the indentation doesn't work and also do you want it to be a subpage?

Reply:
I did want it to be a sub page, but I am not quite happy with that location either ;).  I could not find a "perfect spot" to put this.   It is not practice area specific.

Reply:
My commit has 4 spaces in from the parent (just like all the others) but then some linter removes two of them which makes it fail.

Reply:
I am trying an alternate approach. We mind need to change the first file into README.md to make it work.

Reply:
What do you think about the menu now https://civicactions-handbook--1551.org.readthedocs.build/en/1551/common-practices-tools/contribution/contrib-first/? If we rename the file contrib-first.md file to README.md then it will collapse the double link and the new page will appear underneath it.